README.md

```diff
@@ -180,6 +180,29 @@ when submitting a pull request.
 
 You can run the lint tools manually using `pre-commit run --all`.
 
+## Versioning
```

Comment:
see my comment in BLSQ/openhexa-notebooks#54 about sharing those info in one place

Reply:
Yes, probably a good idea, but I'm adding this to the next documentation todo-list